www/css/album.css

```diff
@@ -0,0 +1,50 @@
+.album-content {
+  margin-left: 15px;
+  margin-right: 15px;
+}
```

Comment:
Move the margin (not `margin-right`) to child containing all the images. See comment in `gallery.css` about the margin and the comment about the changed structure in the album's template.